integration-tests/e2e_tests_ci_tool/cmd/create_test_config_cmd.go

```diff
@@ -0,0 +1,190 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pelletier/go-toml/v2"
+	"github.com/spf13/cobra"
+
+	ctf_config "github.com/smartcontractkit/chainlink-testing-framework/config"
+)
+
+var createTestConfigCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Create a test config from the provided flags",
```

Comment:
So, if I got it right, every time we need to add a new param to the config we need to change it in several places + in CI workflow? Validation is cool but the cost of maintenance, why can't we do "bypass" and insert `base64`? We decided to use TOML everywhere and `base64` to override anything and now we have 2 additional layers to pass all the vars. Maybe if we wait for secrets migration we can do "bypass", wdyt?

Reply:
@skudasov after we remove all secrets from base64ConfigOverride then you will be able to pass it as workflow input. But, even in that case you still want some workflows to have custom inputs like [.github/workflows/run-automation-ondemand-e2e-tests.yml](https://github.com/smartcontractkit/chainlink/pull/13740/files#diff-5c80e74551ba794eb079756afb185869d0f989d1ede2e549fe257f25adc4853e). This workflow does not need full config override. It only needs few inputs related to the specific tests it executes. So it should stay the way it is (the way it was defined by Anirudh).

I also agree that once we don't have secrets in TOML we can simplify .github/actions/create-default-e2e-config-override/action.yml a lot and allow you to pass/override any test input without needing to update this github action.

Reply:
hmmm... if I understand it correctly, then the answer to your question @skudasov is **no**. If you add a new parameter to TOML config you would need to edit workflow/tool **only if** you want this parameter to be automatically populated based on some dynamic logic (like the SHA used). If it's a regular parameter that you need to change only sometimes, you can do it by passing it in a base64 config. The tool @lukaszcl added doesn't need to know anything about this field in that case.

@lukaszcl am I understanding it right?